config/database.yml

```diff
@@ -3,13 +3,13 @@ development:
   encoding: unicode
   database: myflix_development
   pool: 5
-  username: YOUR_USERNAME
-  password: YOUR_PASSWORD
+  username: vincent
+  password: xc1946sjc
```

Comment:
This file is supposed to be hidden, in `.gitignore` and you should _never_ hardcode your password to the app, this is public place, everybody can see it now. I would recommend changing the password now...